example/coffee/models.py

```diff
@@ -21,6 +21,11 @@ def __str__(self):
         return self.name
 
 
+class Favorite(models.Model):
```

Comment:
Open to another way to make the example show updating grand parents, grand children, etc, this was my first quick idea, but it does introduce a little bit of a silly Model.

Reply:
I think the `Favorite` model is reasonable enough. I did need to put a check into the code for data that doesn't have a favorite attached to it and will push that soon.